### PR TITLE
Do strip_large_tensor recursively

### PR DESCRIPTION
**Pull request by impactaky**

### CODE REVIEW

**Comment by kmaehashi** [comment]
LGTM!
Could you take part in the review @take-cheeze? :pray:

File: pytorch_pfn_extras/onnx/strip_large_tensor.py [hunk 1]
```diff
@@ -47,12 +47,20 @@ def _strip_raw_data(tensor):
...
-    for init in onnx_model.graph.initializer:
+def _strip_large_initializer_raw_data_from_graph(graph, large_tensor_threshold):
+    for init in graph.initializer:
         if _is_stripped(init):
             continue
         if is_large_tensor(init, large_tensor_threshold):
             _strip_raw_data(init)
+    for node in graph.node:
+        for attr in node.attribute:
+            if attr.name == 'subgraph':
```
Comment (take-cheeze) [inline pytorch_pfn_extras/onnx/strip_large_tensor.py:58]: Maybe this is more generic:
```suggestion
            if attr.HasField('g'):
```

**Review by take-cheeze — APPROVED** [review]

**Pull request merged**